### PR TITLE
[E2E][ADD] UiDiff on checkboxes

### PR DESCRIPTION
**Pull request by MaximeGenevier**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

File: e2e/DateSelect.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@ui-diff
+Feature: DateSelect
+
+    Background: Specify storybook forms date select
+        Given storybook forms date select
+
+    Scenario: DateSelect-01: I can display date select
+		Then take screenshot
```
Comment (shajz) [inline e2e/DateSelect.feature:8]:
```suggestion
		    Then take screenshot
```
😁

File: e2e/DateSelect.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+    Scenario: DateSelect-02: I can open date select
+		When click on select
+		Then take screenshot
```
Comment (shajz) [inline e2e/DateSelect.feature:12]:
```suggestion
		    When click on select
		    Then take screenshot
```
😁

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

**Review by shajz — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1797/storybook

**Pull request merged**